modules/msk/genomenexus/annotationpipeline/main.nf

```diff
@@ -22,11 +22,11 @@ process GENOMENEXUS_ANNOTATIONPIPELINE {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     """
-    java -Xms${task.memory.toMega()/4}m -Xmx${task.memory.toGiga()}g -jar /genome-nexus-annotation-pipeline/annotationPipeline/target/annotationPipeline.jar --filename ${input_maf} --output-filename ${meta.id}_annotated.maf
```

Comment:
This looks like a regression. I remember we added the java args for a bug fix. Also, it is moving the image version from 1.05 to 1.0.4